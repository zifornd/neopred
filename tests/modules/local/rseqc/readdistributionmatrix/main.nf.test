nextflow_process {

    name "Test Process RSEQC_READDISTRIBUTIONMATRIX"
    script "modules/local/rseqc/readdistributionmatrix/main.nf"
    process "RSEQC_READDISTRIBUTIONMATRIX"

    test("Should run without failures") {

        when {
            params {
                // define parameters here. Example:
                // outdir = "tests/results"
            }
            process {
                """
                // define inputs of the process here. Example:
                input[0] = file("/home/suhail/neopred/.nf-test/tests/9b31d8acefbb9eca2aed6a1e57bef5ab/work/91/54906c99000790215de4eb5f25cc14/test.read_distribution.txt")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
