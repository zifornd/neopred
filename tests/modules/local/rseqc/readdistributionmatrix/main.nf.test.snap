{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "read_distrib.matrix.tab:md5,ec83ce468f8bfebe9d8a4164c61a697b"
                ],
                "1": [
                    "versions.yml:md5,87b14685f09de5edcd7432c89669de3e"
                ],
                "matrix_tab": [
                    "read_distrib.matrix.tab:md5,ec83ce468f8bfebe9d8a4164c61a697b"
                ],
                "versions": [
                    "versions.yml:md5,87b14685f09de5edcd7432c89669de3e"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.1"
        },
        "timestamp": "2024-12-18T15:50:05.472611"
    }
}